Adding eth_getBlockReceipts and Batch request to rpc-support.md (#887)
moonbeam-foundation/moonbeam-docs: builders/get-started/eth-compare/rpc-support.md

- **[eth_getStorageAt](https://eth.wiki/json-rpc/API#eth_getstorageat){target=\_blank}** — returns content of the storage at a given address
- **[eth_getBlockByHash](https://eth.wiki/json-rpc/API#eth_getblockbyhash){target=\_blank}** — returns information about the block of the given hash including `baseFeePerGas` on post-London blocks
- **[eth_getBlockByNumber](https://eth.wiki/json-rpc/API#eth_getblockbynumber){target=\_blank}** — returns information about the block specified by block number including `baseFeePerGas` on post-London blocks

> - **[eth_getBlockReceipts](https://docs.alchemy.com/reference/eth-getblockreceipts){target=\_blank}** — returns all transaction receipts for a given block.

Reviewer: ```suggestion
- **[eth_getBlockReceipts](https://docs.alchemy.com/reference/eth-getblockreceipts){target=\_blank}** — returns all transaction receipts for a given block
```

- **[txpool_status](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool#txpool-status){target=\_blank}** - returns the total number of transactions currently pending, waiting to be included in the next block(s), and queued for future execution

For a dedicated tutorial for these debug and trace methods, check out the [Debug API & Trace Module](/builders/build/eth-api/debug-trace/){target=\_blank} guide.

> ## Batch request {: #batch-request }

Reviewer: ```suggestion
## Batch Requests {: #batch-request }
```

> Moonbeam clients support [Batch request](https://geth.ethereum.org/docs/interacting-with-geth/rpc/batch){target=\_blank}, allowing you to send multiple data requests simultaneously in an array. This reduces network delays by fetching data in a single call, especially beneficial for retrieving large amounts of independent data.

Reviewer: ```suggestion
Moonbeam clients support [batch requests](https://geth.ethereum.org/docs/interacting-with-geth/rpc/batch){target=\_blank}, allowing you to send multiple data requests simultaneously in an array. This reduces network delays by fetching data in a single call, which is especially beneficial for retrieving large amounts of independent data.
```
